test/test_math_ops.cpp

```diff
@@ -1677,6 +1677,122 @@ TEST(MathOps, SquaredDifferenceBroadcasting) {
   opexecuter.RunTest();
 }  // end of test op SquaredDifferenceBroadcasting
 
+// Test op: UnsortedSegmentSum
+TEST(MathOps, UnsortedSegmentSum) {
+  Scope root = Scope::NewRootScope();
+  Tensor A(DT_FLOAT, TensorShape({3, 4}));
+  Tensor B(DT_INT32, TensorShape({3}));
+  Tensor C(DT_INT32, TensorShape({}));
+
+  AssignInputValues(A, std::vector<float>{1.f, 2.f, 3.f, 4.f, 5.f, 6.f, 7.f,
+                                          8.f, 4.f, 3.f, 2.f, 1.f});
+  AssignInputValues(B, std::vector<int>{0, 1, 0});
+  AssignInputValues(C, 2);
+
+  vector<int> static_input_indexes = {1, 2};
```

Comment:
please change the static_input_indexes here also

Reply:
@gopoka Please change the static_input_indexes

Reply:
Done.